homeassistant/components/lyric/strings.json

```diff
@@ -0,0 +1,17 @@
+{
+  "title": "Honeywell Lyric",
```

Comment:
```suggestion
```

The one from `manifest.json` will be used so you can delete this since its the same